examples/assets/ui/welcome.ron

```diff
@@ -0,0 +1,25 @@
+#![enable(implicit_some)]
+Container(
+    transform: (
+        id: "background",
+        anchor: Middle,
+        stretch: XY( x_margin: 0., y_margin: 0., keep_aspect_ratio: false),
+        width: 20.,
+        height: 20.,
+    ),
+
+    children: [
+
+        Image(
+            transform: (
+                id: "background",
+                // y: 540.,            // <- comment this
```

Comment:
I am confused by this line.

Reply:
ah, that was there from a previous version. not sure where the other comment came from.